pkg/driver/node_windows.go

```diff
@@ -49,10 +50,10 @@ func (d *nodeService) findDevicePath(devicePath, volumeID, _ string) (string, er
 
 	foundDiskNumber := ""
 	for diskNumber, diskID := range diskIDs {
-		serialNumber := diskID.Identifiers["serialNumber"]
+		serialNumber := diskID.GetSerialNumber()
 		cleanVolumeID := strings.ReplaceAll(volumeID, "-", "")
 		if strings.Contains(serialNumber, cleanVolumeID) {
-			foundDiskNumber = diskNumber
+			foundDiskNumber = strconv.Itoa(int(diskNumber))
```

Comment:
Was the prior behavior broken?

Reply:
no

Reply:
previously the map was map[string], now it's map[uint32]